github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -25,11 +26,180 @@ presubmits:
         type: bare-metal-external
       containers:
         - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
+          env:
+            - name: TARGET
+              value: k8s-1.20
+            - name: KUBEVIRT_E2E_SKIP
+              value: SRIOV|GPU|Operator|sig-storage|sig-network|sig-compute
```

Comment:
see please k8s-1.19
need quotes and to use '\\[' ' \\]'

Reply:
This match the same tests.

Reply:
We discussed in the past that we should prefect match including the brackets.
I think we should align it to the other jobs.

Reply:
Done